src/Commands/AdminCommands/ChatsCommand.php

```diff
@@ -58,6 +58,7 @@ public function execute()
         $results = DB::selectChats(
```

Comment:
@jacklul Have made a PR to your PR that implements the solution with the array, please check it out when you have a moment, thanks.